src/Form/FormAnalyzer.js

```diff
@@ -117,6 +117,12 @@ class FormAnalyzer {
         shouldCheckUnifiedForm = false,
         shouldBeConservative = false
     }) {
+        // If the string is empty or too long (noisy) do nothing
+        if (
+            !string ||
+            string.length > constants.TEXT_LENGTH_CUTOFF
+        ) return this
+
```

Comment:
This is the meat of the fix. We want to avoid running several regex checks on potentially huge chunks of text. This is what was causing the issues.